Homepage wget error

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Homepage

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/homepage.sh)"

### 📝 Provide a clear and concise description of the issue.

Hi,
I am currently getting the following error with the homepage script:

```
⠴ Updating Homepage to v1.1.1 (Patience)
[ERROR] in line 43: exit code 0: while executing command wget -q https://github.com/gethomepage/homepage/archive/refs/tags/v${RELEASE}.tar.gz
```

Could it be related to this? #3455 

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

update command

### ❌ Paste the full error output (if available).

⠴ Updating Homepage to v1.1.1 (Patience)
[ERROR] in line 43: exit code 0: while executing command wget -q https://github.com/gethomepage/homepage/archive/refs/tags/v${RELEASE}.tar.gz

### 🖼️ Additional context (optional).

_No response_

--- a/ct/homepage.sh
+++ b/ct/homepage.sh
@@ -40,7 +40,7 @@ function update_script() {
   if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
     msg_info "Updating Homepage to v${RELEASE} (Patience)"
     systemctl stop homepage
-    wget -q https://github.com/gethomepage/homepage/archive/refs/tags/v${RELEASE}.tar.gz
+    curl -fsSL "https://github.com/gethomepage/homepage/archive/refs/tags/v${RELEASE}.tar.gz" -o $(basename "https://github.com/gethomepage/homepage/archive/refs/tags/v${RELEASE}.tar.gz")
     tar -xzf v${RELEASE}.tar.gz
     rm -rf v${RELEASE}.tar.gz
     cp -r homepage-${RELEASE}/* /opt/homepage/

--- a/install/homepage-install.sh
+++ b/install/homepage-install.sh
@@ -32,7 +32,7 @@ msg_ok "Installed Node.js"
 LOCAL_IP=$(hostname -I | awk '{print $1}')
 RELEASE=$(curl -s https://api.github.com/repos/gethomepage/homepage/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
 msg_info "Installing Homepage v${RELEASE} (Patience)"
-wget -q https://github.com/gethomepage/homepage/archive/refs/tags/v${RELEASE}.tar.gz
+curl -fsSL "https://github.com/gethomepage/homepage/archive/refs/tags/v${RELEASE}.tar.gz" -o $(basename "https://github.com/gethomepage/homepage/archive/refs/tags/v${RELEASE}.tar.gz")
 $STD tar -xzf v${RELEASE}.tar.gz
 rm -rf v${RELEASE}.tar.gz
 mkdir -p /opt/homepage/config

